src/OpenTelemetry.Instrumentation.AspNetCore/OpenTelemetry.Instrumentation.AspNetCore.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;netstandard2.1</TargetFrameworks>
```

Comment:
@JamesNK We're now targeting `netstandard2.1` and conditionally compiling the gRPC functionality for this instrumentation. However, I did note that the Grpc.AspNetCore library targets `netcoreapp3.0`. 
https://github.com/grpc/grpc-dotnet/blob/ef77760676ce19c279d5480bca68af3ea4d60e96/src/Grpc.AspNetCore/Grpc.AspNetCore.csproj#L8

Would it make more sense for us to target `netcoreapp3.0` as well?

Reply:
netcoreapp3.0 supports netstandard2.1. It doesn't matter either way.

Reply:
Ok, thanks! That's what I figured, but just wanted to be sure.